terraform/create_tf_vars.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
The purpose of this script is to easily generate a valid `variables.tfvars` file for local Terraform deployments or tests.